docs/api/native-image.md

```diff
@@ -74,72 +74,80 @@ mixed with other content to create the desired final appearance.
 The most common case is to use template image for menu bar icon so it can adapt
 to both light and dark menu bars.
 
-Template image is only supported on Mac.
+Template image is only supported on OS X.
 
-To mark an image as template image, its filename should end with the word
+To mark an image as a template image, its filename should end with the word
 `Template`, examples are:
 
 * `xxxTemplate.png`
 * `xxxTemplate@2x.png`
 
-## nativeImage.createEmpty()
+## Methods
+
+The `NativeImage` class has the following methods:
+
+### `NativeImage.createEmpty()`
 
 Creates an empty `NativeImage` instance.
 
-## nativeImage.createFromPath(path)
+### `NativeImage.createFromPath(path)`
 
 * `path` String
 
 Creates a new `NativeImage` instance from a file located at `path`.
 
-## nativeImage.createFromBuffer(buffer[, scaleFactor])
+### `NativeImage.createFromBuffer(buffer[, scaleFactor])`
 
 * `buffer` [Buffer][buffer]
-* `scaleFactor` Double
+* `scaleFactor` Double (optional)
 
-Creates a new `NativeImage` instance from `buffer`. The `scaleFactor` is 1.0 by
-default.
+Creates a new `NativeImage` instance from `buffer`. The default `scaleFactor` is
+1.0.
 
-## nativeImage.createFromDataUrl(dataUrl)
+### `NativeImage.createFromDataUrl(dataUrl)`
 
 * `dataUrl` String
 
 Creates a new `NativeImage` instance from `dataUrl`.
 
-## Class: NativeImage
+## Instance Methods
```

Comment:
Double checking this Instance Methods title is :ok: @zcbenz as well as the change of the method headers to `image.` an instance described on `118`. This is how we did it for `BrowserWindow` with `win`.